agentlocal/agentlocal.go

```diff
@@ -55,6 +55,7 @@ const (
 
 // ErrNotSetUp is returned by GetStatus when pmm-agent is running, but not set up.
 var ErrNotSetUp = fmt.Errorf("pmm-agent is running, but not set up")
+var ErrNotConnected = fmt.Errorf("pmm-agent is not connected to PMM server")
```

Comment:
Looks like it is used only once.

Reply:
Yes, it is. But it's extracted to make it visible.